app/ember/points/Home.tsx

```diff
@@ -1,274 +1,73 @@
-      point: "300 Points",
-      frequency: "One-time",
-      image: artrific,
-      buttons: [
-        {
-          text: "Go to Quest",
-          link: "https://app.layer3.xyz/quests/explore-artrific-nft-marketplace-on-fuse-network",
-        },
-        {
-          text: "Verify Quest",
-          isFunction: true,
-        }
-      ]
-    },
-    {
-      id: "g$Claim",
-      title: "Claim G$ on GoodDapp",
-      description: "To get 30 points daily, you need to take 6 steps:  \n**Step 1:**\nGo to quest on the Layer3 platform  \n**Step 2:**\nConnect to Layer3 a wallet participating in the airdrop  \n**Step 3:**\nGo to GoodDapp  \n**Step 4:**\nClaim G$ token on Fuse Network  \n**Step 5:**\nVerify quest completion on the Layer3  \n**Step 6:**\nRepeat every day. After 5 claims, the quest will renew automatically and allow you to claim more and more.  \n**Verify the quest 1 hour after completing it on Layer3**",
-      point: "30 Points per claim",
-      frequency: "Once a day",
-      image: gooddollarYellow,
-      buttons: [
-        {
-          text: "Go to Quest",
-          link: "https://app.layer3.xyz/streaks/claim-dollarg",
-        },
-        {
-          text: "Verify Quest",
-          isFunction: true,
-          endpoint: "gooddollar",
-        }
-      ]
-    },
-  ])
+const Home = () => {
+  const {user} = useAppSelector(selectAirdropSlice);
+  const {isConnected} = useAccount();
+  const [email, setEmail] = useState("");
+  const [subscriptionStatus, setSubscriptionStatus] = useState<
+    "idle" | "loading" | "success" | "error"
+  >("idle");
+  const [subscriptionMessage, setSubscriptionMessage] = useState("");
 
-  function referralLink() {
-    const host = !IS_SERVER ? window?.location?.host : ""
-    return `${host}/ember?ref=${user.referralCode}`
-  }
+  const handleSubscribe = async (e: React.FormEvent) => {
+    e.preventDefault();
+    if (!email || !email.includes("@")) return;
 
-  useEffect(() => {
-    setQuests((prevQuests) => {
-      const newQuests = [...prevQuests];
-      newQuests.map((newQuest) => {
-        if (newQuest.frequency !== "One-time") {
-          return newQuest;
-        }
+    setSubscriptionStatus("loading");
 
-        user.completedQuests?.map((completedQuest) => {
-          let completedQuestId = completedQuest.type;
-          if (completedQuest.stakingType) {
-            completedQuestId = `${completedQuest.type}-${completedQuest.stakingType}`;
-          }
-          if (newQuest.id === completedQuestId) {
-            newQuest.completed = true;
-          }
-        })
-        return newQuest;
-      });
-      return newQuests;
-    })
-  }, [user.completedQuests])
+    try {
+      const iframe = document.createElement("iframe");
+      iframe.style.display = "none";
+      iframe.src = `https://magic.beehiiv.com/v1/e8385c43-f27c-410c-93de-7f07b6492818?email=${encodeURIComponent(
```

Comment:
@NikolayRn is this working? Seems like email is not sent to Beehiiv:
https://app.beehiiv.com/subscribers